docs/source/usage/tbg.rst

```diff
@@ -3,15 +3,30 @@
 TBG
 ===
 
-.. sectionauthor:: Axel Huebl, Richard Pausch
+.. sectionauthor:: Axel Huebl
 .. moduleauthor:: René Widera
 
-todo: explain idea and use case
+Our tool *template batch generator* (``tbg``) abstracts program runtime options from technical details of supercomputers.
+On a desktop PC, one can just execute a command interactively and instantaneously.
+Contrarily on a supercomputer, resources need to be shared between different users efficiently via *job scheduling*.
+Scheduling on today's supercomputers is usually done via *batch systems* that define various queues of resources.
+
+An unfortunate aspect about batch systems from a user's perspective is, that their usage varies a lot.
+And naturally, different systems have different resources in queues that need to be described.
+
+PIConGPU runtime options are described in *configuration files* (``.cfg``).
+We abstract the description of queues, resource acquisition and job submission via *template files* (``.tpl``).
+For example, a ``.cfg`` file defines how many *devices* shall be used for computation, but a ``.tpl`` file calculates how many *physical nodes* will be requested.
```

Comment:
The `.tpl`file does not calculate  the number of nodes, it defines a distribution which is used by `tbg` to perform a calculation.